README.md

```diff
@@ -1,50 +1,33 @@
+# Packit
+
 [![Build Status](https://zuul-ci.org/gated.svg)](https://softwarefactory-project.io/zuul/t/local/builds?project=packit-service/packit)
 [![black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 
-![Packit](design/export/logo-extended.png)
+![Packit](design/export/logo-no-borders.png)
 
-## Elevator pitch
+---
 
 Packit is a CLI tool that helps developers auto-package upstream projects
 into Fedora operating system.
+
 You can use packit to continously build your upstream project in Fedora.
-With packit you can create SRPMs, open pull requests in dist-git, submit koji builds and even
-create bodhi updates, effectively replacing the whole Fedora packaging workflow.
-
-## Plan and current status
-
-We are working on two things now:
-
-1.  Packit as a tool - a standalone CLI tool which you can install from Fedora
-    repositories and use easily.
-2.  Packit service - A service offering built on top of packit tool. Our
-    expectation is that you would add packit service into your Github
-    repository and it would start handling things automatically: opening pull
-    requests on dist-git, building packages, creating updates, ...
-
-For the run-down of the planned work, please see the task-list below.
-
-- [ ] E2E workflow for getting upstream releases into Fedora using packit CLI.
-  - [x] Bring new upstream releases into Fedora rawhide as dist-git pull
-        requests. ([propose-update](https://packit.dev/docs/cli/propose-update/) command included in 0.1.0 release)
-  - [x] Build the change once it's merged. #137
-  - [x] Send new downstream changes back to upstream. (so the spec files are in
-        sync) #145
-  - [x] Packit can create bodhi updates. #139
-  - [x] Ability to propose updates also to stable releases of Fedora.
-  - [x] Create SRPMs from the upstream repository
-  - [x] Build RPMs in COPR and integrate the results into Github.
-- [ ] source-git
-  - [x] Packit can create a SRPM from a source-git repo.
-  - [ ] You can release to rawhide from source-git using packit.
-  - [ ] Packit can create a source-git repository.
-  - [ ] Packit helps developers with their source-git repositories.
-- [x] Packit as a service
-  - [x] Packit reacts to Github webhooks.
-  - [x] Have a Github app for packit.
-    - [x] Github app is on Marketplace.
-  - [x] Packit service is deployed and usable by anyone.
+
+With packit you can create SRPMs, open pull requests in dist-git, submit koji builds and even create bodhi updates, effectively replacing the whole Fedora packaging workflow.
+
+---
+
+## To start using Packit
+
+See our documentation on [packit.dev]
+
+Learn more on [Packit as a service](https://packit.dev/docs/packit-as-a-service/)
+
+Try Packit [Testing Farm](https://packit.dev/docs/testing-farm/)
```

Comment:
I think we don't need the links to `Packit as a service` and `Testing Farm` here. They are on the website and this readme is just for `packit`.

Reply:
I added them to make it faster **to get these information for a newcomer**.  
But I can remove them, no problem.